bigtable-client-core-parent/bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/BigtableSession.java

```diff
@@ -198,6 +199,9 @@ private synchronized static void initializeResourceLimiter(BigtableOptions optio
    */
   private BigtableClusterName clusterName;
 
+  private ThrottlingClientInterceptor throttlingClientInterceptor =
+      new ThrottlingClientInterceptor();
+
```

Comment:
The interceptor is now added to the array of interceptors in BigtableSession's constructor.